packages/discord.js/src/client/websocket/WebSocketManager.js

```diff
@@ -318,7 +318,7 @@ class WebSocketManager extends EventEmitter {
    */
   destroy() {
     if (this.destroyed) return;
-    this.debug(`Manager was destroyed. Called by:\n${new Error('MANAGER_DESTROYED').stack}`);
+    this.debug(`Manager was destroyed. Called by:\n${new globalThis.Error().stack}`);
```

Comment:
We should probably make this an utility or something, and the message becomes a bit useless here, as it includes an `Error` line (the error class name + message if any) followed by the stack.

If it's not done in this PR, it can be done in a follow-up one, and a `TODO` can be written here.